### PR TITLE
Add old cpfrontend host to ingress

### PR DESCRIPTION
**Pull request by andyhd**
This will make the new Control Panel receive requests to `https://cpanel-master.services...`

I'm assuming the way to actually remove the old frontend is to just `helm delete` it, but let me know if you have a nicer way

### CODE REVIEW

File: charts/cpanel/templates/ingress.yaml [hunk 1]
```diff
@@ -22,6 +22,12 @@ spec:
       - backend:
           serviceName: "{{ .Chart.Name }}"
           servicePort: 80
+  - host: {{ template "old_cpfrontend_host" . }}
```
Comment (xoen) [inline charts/cpanel/templates/ingress.yaml]: Where is this defined?

**Comment by xoen** [comment]
I thought new CP URL would be `controlpanel.{{ Values.servicesDomain }}`? Maybe I got confused.

**Comment by andyhd** [comment]
Yes it is `controlpanel.{{ .Values.serviceDomain }}` - this is so that the old URL points to the new Control Panel

**Comment by r4vi** [comment]
will links to assets like: https://cpanel-master.services.alpha.mojanalytics.xyz/static/images/gov.uk_logotype_crown.svg
used in https://github.com/ministryofjustice/analytics-platform-auth0/blob/alpha/pages/login.html#L146 continue to work?

**Comment by andyhd** [comment]
Ah, no. Should I make that work or should we get the new URL into the login page template?

**Comment by r4vi** [comment]
@andyhd what's easier? it's pretty easy to update those templates but I wonder where else it's being used

**Comment by andyhd** [comment]
[Technically, we shouldn't be using the crown logo anyway.](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk) I don't know what else is deep-linking into the Control Panel frontend. Probably just the login page. The new Control Panel uses the GOV.UK Design System, which may well have different image assets from the old GOV.UK Frontend and Elements stuff too.

**Comment by andyhd** [comment]
Probably [one of these](https://www.google.com/search?q=moj+logo&tbm=isch)

**Comment by r4vi** [comment]
this is nice

**Pull request merged**